packages/run-it/src/components/RequestForm/RequestForm.spec.tsx

```diff
@@ -174,7 +174,7 @@ describe('RequestForm', () => {
     )
 
     const item = screen.getByRole('switch', { name })
-    userEvent.click(item)
+    await userEvent.click(item)
```

Comment:
Get rid of the following waitFor now?

Reply:
Had to add the `waitFor` back because the test was failing without it.